ted_data_eu/adapters/storage.py

```diff
@@ -0,0 +1,90 @@
+import json
+from typing import Dict, List
+
+import requests
+from requests import Response
+from requests.auth import HTTPBasicAuth
+
+from ted_data_eu import config
+from ted_data_eu.adapters.storage_abc import DocumentStorageABC
+
+BASIC_HEADERS = {'Content-type': 'application/x-ndjson'}
+
+
+class ElasticStorage(DocumentStorageABC):
+    """
+       Implements interaction with ElasticSearch storage by using its API.
+    """
+
+    def __init__(self,
+                 index: str,
+                 host: str = None,
+                 user: str = None,
+                 password: str = None):
+        """
+           Implements interaction with ElasticSearch storage by using its API.
+
+            :param index: ElasticSearch index (schema) where the documents will be stored
+            :param host: Elastic API host and port (if different from the defaults for http and https)
+            :param user: Elastic API user
+            :param password: Elastic API password
+            :return:
+        """
+        self.host = host if host else config.ELASTIC_HOST
```

Comment:
self.host = host or config.ELASTIC_HOST

Also for similar inline ifs

it is more compact